misc/python/materialize/lint/lint.py

```diff
@@ -38,7 +38,8 @@ def main() -> int:
     verbose_output = args.verbose
 
     manager = LintManager(print_duration, verbose_output)
-    return manager.run()
+    return_code = manager.run()
+    exit(return_code)
```

Comment:
I think it would be cleaner to call `exit(main())` at the end of the file.

Reply:
✅ Done.

Shall I do that as follow-up in other files as well?

Reply:
Yes, would be nice. Functions really shouldn't call exit, otherwise you can't use them in other parts of the code.

Reply:
Sorry, my question was ambiguous. Shall I make sure that `exit` is included in all other blocks of type

```
if __name__ == "__main__":
    main()
```
?

Reply:
If it returns a code yes. If not no.